data/postprocess/velocity-residual/point_values_velocity.txt

```diff
@@ -0,0 +1,7 @@
+# Test data for ascii data boundary velocity.
```

Comment:
Update this text. Do you want to use a nonzero velocity here?

Reply:
I used zero values so that we can compare the velocities using `point_values` postprocessor (they should be the same). I have updated the description now so that it makes sense, let me know if you have another idea.